mail_tracking/static/src/xml/failed_message.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+
+    <t t-name="mail_tracking.failed_message_items">
+        <div class="o_thread_date_separator o_border_dashed" data-toggle="collapse" data-target="#o_chatter_failed_message">
+            <a role="button" class="o_thread_date btn">
+                <i class="fa fa-fw fa-caret-down"/>
+                Failed messages
+                <small class="o_chatter_failed_message_summary ml8">
+                    <span class="badge rounded-circle badge-danger"><t t-esc="nbFailedMessages"/></span>
+                </small>
+            </a>
+        </div>
+        <div id="o_chatter_failed_message" class="collapse in show">
+            <t t-foreach="failed_messages" t-as="message">
+                <div class="o_thread_message" style="margin-bottom: 10px">
+                    <div class="o_thread_message_sidebar">
+                        <div class="o_avatar_stack">
+                            <img t-attf-src="/web/image#{message.author_id[0] >= 0 ? ('/res.partner/' + message.author_id[0] + '/image_small') : ''}" class="o_thread_message_avatar rounded-circle mb8" t-att-title="message.author_id[1]" t-att-alt="message.author_id[1]"/>
+                            <i t-att-class="'o_avatar_icon fa fa-exclamation bg-danger-full'"
+                               title="Failed"/>
+                        </div>
+                    </div>
+                    <div class="o_thread_message_core">
+                        <div class="o_mail_info text-muted">
+                            <strong class="o_thread_author">
+                                <t t-esc="message.author_id[1]"/>
+                            </strong>
+                            - <small class="o_mail_timestamp" t-att-title="message.date.format(date_format)"><t t-esc="message.hour"/></small>
+                            <span t-attf-class="o_thread_icons">
+                                <a href="#" class="btn btn-link btn-success text-muted btn-sm o_failed_message_reviewed o_activity_link" t-att-data-message-id="message.id">
+                                    <i class="fa fa-check"/> Mark Reviewed
+                                </a>
+                                <a href="#" class="btn btn-link btn-default text-muted btn-sm o_failed_message_retry" t-att-data-message-id="message.id">
+                                    <i class="fa fa-retweet"/> Retry
+                                </a>
+                            </span>
+                            <br/>
+                            <strong class="text-danger">Failed Recipients:</strong>
+                            <t t-set="first_recipient" t-value="true"/>
+                            <t t-foreach="message.failed_recipients" t-as="recipient">
+                                <t t-if="!first_recipient">
```

Comment:
Qweb automatically adds this variable inside loops: 

```suggestion
                                <t t-if="!recipient_first">
```

Docs: https://www.odoo.com/documentation/12.0/reference/qweb.html#loops